README.md

```diff
@@ -4,11 +4,6 @@ The **Azure IoT Edge Dev Tool** greatly simplifies [Azure IoT Edge](https:/azure
 
  - It gets you started with IoT Edge development with the [IoT Edge Dev Container](#iot-edge-dev-container) and IoT Edge Solution Scaffolding that contains a sample module and all the required configuration files.
  - It speeds up your inner-loop dev (dev, debug, test) by reducing multi-step build & deploy processes into one-line CLI commands and well as drive your outer-loop CI/CD pipeline. _You can use all the same commands in both stages of your development life-cycle._
```

Comment:
I feel like we need a link to the wiki at the top, maybe after the intro.  Something like....for full docs see the wiki, link.

Reply:
Agreed, that's what I initially had but then coupled it with contributing.md in resources. Want to split that back out or leave the link there and add another reference at the top?

Reply:
Let's add another at top...just not under resources header...probably in overview section would be best.